en/04_Changelogs/5.0.0.md

```diff
@@ -26,6 +28,53 @@ title: 5.0.0 (unreleased)
   - [Other features](#other-features)
 - [Bugfixes](#bugfixes)
 
+## Change to supported modules {#supported-modules}
+
+Some Silverstripe CMS modules are commercially supported. Silverstripe commits to looking after those modules for the duration of the Silverstripe CMS 5 lifecycle.
+
+Review the list of [Silverstripe CMS 5 supported modules](/Project_Governance/Supported_Modules).
+
+### Module not supported in Silverstripe CMS 5
+Some modules that were supported in Silverstripe CMS 4 will not be supported in Silverstripe CMS 5. Just because a module is not supported officially, doesn't mean that you shouldn't be using it. However, it does mean that it's more likely to suffer from regressions. You should take this in consideration when choosing to install an unsupported module in your project.
```

Comment:
I think saying that it's more likely modules will suffer from regressions might be scaring people unnecessarily. Can we find better wording? Because if no one touches the code then how would a regression occur?

Reply:
Even if no one touches the code, it is still possible that later change in a CMS 5 minor will break things.

I'm happy to rethink this wording, we shouldn't say that running unsupported module is just a safe as running supported ones.

Reply:
I think it does the community an injustice to effectively say "all community modules are less safe than the ones _we_ look after"